src/HeadersParser.h

```diff
@@ -243,12 +245,6 @@ namespace snowcrash {
                         headerSM.sourceMap = node->sourceMap;
                         out.sourceMap.collection.push_back(headerSM);
                     }
-                } else {
-                    // WARN: unable to parse header
-                    mdp::CharactersRangeSet sourceMap = mdp::BytesRangeSetToCharactersRangeSet(node->sourceMap, pd.sourceCharacterIndex);
```

Comment:
Why did we remove this? What happens when [this](https://github.com/apiaryio/snowcrash/pull/412/files#diff-f163723f2e86fab9b0d1f3e1e1a5e992R204) statement runs?

Can you add a test for that?